packages/catana/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
```

Comment:
This not the version returned by `pkgver()`, checkout our templates. https://github.com/BlackArch/blackarch-pkgbuilds

Reply:
Changed in commit 550e72f80ebc3d7322f2ceed2d2f737ad3494375